src/webgl/context-state.js

```diff
@@ -535,26 +536,15 @@ export function setParameter(gl, key, value) {
 export function withState(gl, params, func) {
   // assertWebGLContext(gl);
   const state = getState(gl);
-
```

Comment:
While I agree with your reasoning, I am a little worried that there are places within the code that might be affected.

I should probably just prioritize a fix in that does this the right way. Can you comment it out for now and add a TODO with my name?